server/api/cvrs.py

```diff
@@ -249,6 +249,12 @@ def parse_clearballot_cvrs(
     cvr_file = retrieve_file(jurisdiction.cvr_file.storage_path)
     cvrs = csv_reader_for_cvr(cvr_file)
     headers = next(cvrs)
+
+    if not any(header.startswith("Choice_") for header in headers):
+        raise UserError(
+            "CVR file should have at least one column beginning with 'Choice_'"
+        )
+
     first_contest_column = next(
         i for i, header in enumerate(headers) if header.startswith("Choice_")
     )
```

Comment:
`next` can take a default, so another option here might be:

```python
    first_contest_column = next(
        i for i, header in enumerate(headers) if header.startswith("Choice_"),
        None
    )
    if first_contest_column is None:
       raise ...
```
The way you have it is totally fine too, just offering

Reply:
Nice! I don't have a strong preference either way, so I'll just keep this as is